TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/timeline/MainTimeline.java

```diff
@@ -57,7 +59,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
     @Override
     protected void loadTweet(boolean isPullLoad, boolean isClickLoad) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 26 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS3776)